client/network/src/protocol_controller.rs

```diff
@@ -659,7 +705,13 @@ impl ProtocolController {
 			return false
 		};
 
-		trace!(target: LOG_TARGET, "Peer {peer_id} ({direction:?}) dropped.");
+		trace!(
+			target: LOG_TARGET,
+			"Peer {} ({:?}) dropped from {:?}.",
+			peer_id,
+			direction,
```

Comment:
```suggestion
			"Peer {peer_id} ({direction:?}) dropped from {:?}.",
```

Not sure why you reverted it here and at most of the other places here in this pr?